exec/java-exec/src/main/java/org/apache/drill/exec/ops/OperatorContext.java

```diff
@@ -43,10 +45,18 @@
 
   public abstract OperatorStats getStats();
 
+  public abstract ExecutorService getExecutor();
+
+  public abstract ExecutorService getScanExecutor();
```

Comment:
I kept it like this to minimize touching parts of the code that have nothing to do with the Parquet reader.